CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
 - [Rolling Pool Update] Don't fail if `load-balancer` plugin is missing (Starter and Enterprise plans)
+- [New sr] Fix `sr.probeNfsExists` with value as "No selected value" [#5991](https://github.com/vatesfr/xen-orchestra/issues/5991)(PR [#6137](https://github.com/vatesfr/xen-orchestra/pull/6137))
```

Comment:
- Capitalize "SR"
- Users don't know what `sr.probeNfsExists` is except if it's in an error message or something like that
- Missing space between issue and PR